geomesa-accumulo/geomesa-accumulo-spark-runtime-accumulo2/pom.xml

```diff
@@ -7,8 +7,8 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <name>GeoMesa Accumulo Spark Runtime</name>
-    <artifactId>geomesa-accumulo-spark-runtime_2.11</artifactId>
+    <name>GeoMesa Accumulo Spark Runtime, Accumulo 2.x</name>
+    <artifactId>geomesa-accumulo-spark-runtime-accumulo2_2.11</artifactId>
```

Comment:
Eh, naming things is hard.  How about 'geomesa-accumulo{1,2}-spark-runtime'?  

Doh, are already doing this for HBase...  So I suppose I'm wondering about changing that as well...

Reply:
i agree the names are a bit unwieldy, but i think this is the clearest way to name them - similar to how we have _2.11 to indicate scala version, we have -accumulo1 to indicate accumulo version, but the core artifact name is the same.